easybuild/easyconfigs/l/libpng/libpng-1.6.21-foss-2015b.eb

```diff
@@ -0,0 +1,19 @@
+easyblock = 'ConfigureMake'
+
+name = 'libpng'
+version = '1.6.21'
+
+homepage = 'http://www.libpng.org/pub/png/libpng.html'
+description = "libpng is the official PNG reference library"
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+toolchainopts = {'pic': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [('zlib', '1.2.8')]
+
+configopts = "--with-pic"
```

Comment:
It's not so much about being strict, but keeping easyconfigs for the same software verison in sync...